core/modules/wdb/ChunkResource.cc

```diff
@@ -196,9 +206,38 @@ class Backend {
         }
         return true;
     }
+
     void discard(ScTableVector const& v) {
         _discard(v.begin(), v.end());
     }
+
+    enum LockStatus {UNLOCKED, LOCKED_OTHER, LOCKED_OURS};
+    std::string strLockStatus(LockStatus ls) {
```

Comment:
Sometimes it's nice to have both (so that you don't have to explicitly create temporary streams if you just want a string or figure out how to convert things to a string if you just have a stream), with one implemented in terms of the other. I'd be happier if this function used a standard name (for us) like `toString`, and I don't understand why it is a non-static member of Backend.